### ISSUE TITLE
[Suggestion] Overgrown Style: Add stylevar / Item menu option, to make glass to be fractured

### ISSUE THREAD

**Issue by LautaroL20** [TeamSpen210/BEE2-items#869]
![20160919162338_1](https://cloud.githubusercontent.com/assets/12244376/18965183/803ce95c-8652-11e6-87b0-0da4f87cede1.jpg)

**Comment by TeamSpen210** [comment]
I need to figure out a good way to do this - those are models premade for one size, we need to have it adapt to the shape of the glass in the map.

**Comment by LautaroL20** [comment]
Yes, the screenshot is purely ilustrative. I maded that prefab glass for my overg chamber to give it a more damaged athmosphere.

**Comment by Flametusk** [comment]
Maybe, have it so that glass in the overgrown style that is the default size and isn't connected to any other glass automatically is fractured?

**Comment by LautaroL20** [comment]
@Flametusk - I think it should be fractured in all the sections.

**Comment by Luke18033** [comment]
There are 128x128 cracked glass models in the base game, just place one on each tile?

**Comment by TeamSpen210** [comment]
Well, the problem with that is that it wouldn't then tile correctly, the cracks wouldn't line up.

**Comment by Luke18033** [comment]
Ah yeah. You'd probably want to make a few custom versions which are tileable.

**Comment by Luke18033** [comment]
Though realistically, if large panes of glass were that cracked they'd probably just shatter. It should probably only apply to small panes of glass, like the one shown in Lautaro's original screenshot.
